src/main/java/codeu/controller/ProfileServlet.java

```diff
@@ -155,10 +144,29 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     }
 
     String aboutMeContent = request.getParameter("About Me");
+    String newHashtagContent = request.getParameter("hashtag");
+
+    if (newHashtagContent != null && newHashtagContent.length() > 0) {
+      String cleanedHashtag = Jsoup.clean(newHashtagContent, Whitelist.none());
+      Hashtag newHashtag =
```

Comment:
I'm thinking we can use the `if (!Util.isNullOrWhitespace(newHashtagContent))` here to avoid empty tags